apps/src/StudioApp.js

```diff
@@ -251,6 +251,15 @@ StudioApp.prototype.init = function(config) {
     config = {};
   }
 
+  if (config.isLegacyShare && config.controllerAction == 'show') {
```

Comment:
What other actions are used on legacy shares? i.e. When does config.isLegacyShare && !config.isLegacyShareShow resolve to true?